DEVELOPMENT.md

```diff
@@ -93,25 +119,66 @@ After which, you can rerun a `px auth login` to authenticate with the dev cloud
 Make sure to `px delete --clobber` if running a prior instance of Vizier pointing to another cloud instance.
 
 ## Vizier
+Vizier is Pixie’s data collector that runs on each cluster. It is responsible
+for query execution and managing PEMs.
+
+### Getting Started
+Verify that you can build Pixie Vizier and run the unit tests.
+
+The following will build Pixie Vizier and then run the unit tests for the UDF
+(User Defined Function) math operations. These UDFs will be available directly off
+the px object inside a PxL script. If you are running the development environment via
+Docker, the following should be run inside the Docker container.
+
+```
+bazel test //src/carnot/funcs/builtins:math_ops_test --test_output=errors -j 2
+```
+
+### Deploying
+Deploying a development version of Vizier is a 2-step process. An official
+release-version of Vizier must first be deployed (through the Pixie CLI or
+YAMLs) and Skaffold can then be run to build and deploy a local development
+version of Vizier.
 
-Deploying a development version of Vizier is a 2-step process. A release-version of Vizier must first be deployed (through the CLI or YAMLs), which will set up specific cluster-secrets that are not deployed via Skaffold. Skaffold can then be run to build and deploy a version of Vizier based on local changes.
+1. Since both of these steps require a Kubernetes cluster to deploy to, you must have
+Minikube installed and running. Follow the instructions
+[here](https://docs.px.dev/installing-pixie/setting-up-k8s/minikube-setup/). Note that
+both deployment scripts below use the output of `kubectl config current-context` to
+determine which Kubernetes cluster to deploy to. So make sure if you have multiple
+clusters, the context is pointing to the correct target cluster.
 
-1. If deploying Vizier pointing to a dev cloud instance, make sure to export the following environment variables:
+1. If you wish to test development changes made to Pixie Cloud and Vizier, export the
+following environment variables that will point to the development Pixie Cloud instance:
 
     ```
     export PL_CLOUD_ADDR=dev.withpixie.dev:443
     export PL_TESTING_ENV=dev
     ```
 
-2. Deploy Vizier through CLI/YAML. If deploying Vizier to a dev cloud instance, make sure to export the necessary dev-specific environment variables mentioned in `Pixie Cloud` above.
+1. Install the Pixie CLI and signup for Pixie Community Cloud. Follow steps 1 and 3 on
```

Comment:
We can't actually tell people to sign up for Community Cloud for Pixie since that would be exclusionary to self-hosted Pixie users. I would tell them to just follow one of the entire guides here: https://docs.px.dev/installing-pixie/install-guides/ (which also includes the `px deploy` step so you can remove it below) and ensure that they can successfullly a script in Pixie either via the UI or CLI.

Reply:
I wasn't sure where to put the extra info here about what `px deploy` does and what to do if it fails so I've combined it into this step here. A more appropriate location might be to actually move this into the docs we link to. If you agree, I'll remove it from this PR and have Elaine add it into her pixie-doc PR she's working on.